### DISCUSSION TITLE
Start using Prisma Client in Node.js

### DISCUSSION THREAD

**Discussion by mr-scrpt** [prisma/prisma#22368] [Q&A] [answered]
### Bug description

I'm trying to run npx prisma generate
But I get an error
<img width="1369" alt="image" src="https://github.com/prisma/prisma/assets/48452376/1a485d92-a65a-4c02-834b-a8b6686838fc">
Until a few weeks ago everything was working fine, but today I created a new model and tried to generate a new client to work with the new model and started getting an error. I tried deleting the new model and creating a minimal model for the test but it doesn't work

### How to reproduce

Just run npx prisma generate

### Expected behavior

It should work.

### Prisma information

```prisma
generator client {
  provider = "prisma-client-js"
}


datasource db {
  provider     = "mysql"
  url          = env("DATABASE_URL")
  relationMode = "prisma"
}

model Store {
  id        String   @id @default(uuid())
  name      String   @unique
  slug      String   @unique
  userId    String
  createdAt DateTime @default(now())
  updateAt  DateTime @updatedAt
  billboards      Billboard[] @relation("StoreToBillboard")
}

model Billboard {
  id          String @id @default(uuid())
  storeId     String
  store       Store @relation("StoreToBillboard", fields: [storeId], references: [id])
  label       String
  imageUrl    String
  createdAt   DateTime @default(now())
  updatedAt   DateTime @updatedAt

  @@index([storeId])
}

```

```ts
import { PrismaClient } from "@prisma/client";

declare global {
  var prisma: PrismaClient | undefined;
}
const prismaDB = globalThis.prisma || new PrismaClient();

if (process.env.NODE_ENV !== "production") globalThis.prisma = prismaDB;

export default prismaDB;

```

### Environment & setup

- OS: macOS Sonoma 14.1.2
- Database: PlanetScale
- Node.js version: 20.9.0

### Prisma Version

```
    "@prisma/client": "5.7.0",
    "prisma": "5.7.0",


```

**Comment by janpio** [comment]
What is the error? The screenshot you shared looks like the expected output of `prisma generate`.

**Comment by mr-scrpt** [comment, marked as answer]
> What is the error? The screenshot you shared looks like the expected output of `prisma generate`.

I apologize, there is now a new model in the prisma client
